### PR TITLE
Article Page - allow Airtable block and Video block & rearrange blocks so they are in alphabetical order

### LINKED ISSUE #6167: [Publications] Airtable Option on Article Pages
We are transferring https://foundation.mozilla.org/en/initiatives/data-futures/who-is-trying/ to a publication template page https://foundation.mozilla.org/en/data-futures-lab/data-for-empowerment/alternative-data-governance-approaches-global-landscape-scan-and-analysis/.

Currently bannered wagtail pages have the airtable embed options available but article pages do not so it would be helpful to include so we can have the airtable present as we move over the long-form contents.

### LINKED ISSUE #6208: [Publications] Video Embed for Article Page
The option to embed videos similar to what we have on wagtail default pages.

### PR DESCRIPTION
**Pull request by mmmavis**
Closes #6208 
Closes #6167

https://foundation-s-issue-6208-egwotq.herokuapp.com/cms/pages/55/edit/
https://foundation-s-issue-6208-egwotq.herokuapp.com/en/publication-page-with-child-article-pages/fixed-title-article-page/

### CODE REVIEW

File: network-api/networkapi/wagtailpages/pagemodels/article_fields.py [hunk 1]
```diff
@@ -6,15 +6,17 @@
...
-        template="wagtailpages/blocks/article_table_block.html"
+    ('content', customblocks.ArticleRichText(
+        features=['bold', 'italic', 'h2', 'h3', 'h4', 'h5', 'link', 'large', 'ol', 'ul', 'image', 'hr', 'footnotes'],
     )),
     ('double_image', customblocks.ArticleDoubleImageBlock()),
     ('full_width_image', customblocks.ArticleFullWidthImageBlock()),
+    ('table', TableBlock(
+        template="wagtailpages/blocks/article_table_block.html"
+    )),
+    ('video', customblocks.VideoBlock()),
```
Comment (mmmavis) [inline network-api/networkapi/wagtailpages/pagemodels/article_fields.py:21]: Added two new options `airtable` and `video`. But I have also rearranged the list in alphabetically.

**Review by Pomax — APPROVED** [review]
I wish this didn't need a migration, but: yay, R+

**Comment by mmmavis** [comment]
@Pomax yea I was surprised that this needed a migration :x

**Review by sabrinang — COMMENTED** [review]
Thanks for organizing block order. 
The video block works fine however I can't get the airtable embed to appear. The height option works as I see the box outline change, we can include in help text this is pixels.
image

**Comment by mmmavis** [comment]
> The video block works fine however I can't get the airtable embed to appear. The height option works as I see the box outline change, we can include in help text this is pixels.

@sabrinang Ah our review apps need to whitelist Airtable. I've fixed it: https://foundation-s-issue-6208-s0aeff.herokuapp.com/en/publication-page-with-child-article-pages/fixed-title-article-page/

Since the Airtable block is a shared block across different page type and that this PR is about enabling the block on Article page, updating the Airtable block itself is out of scope. Can you file a new ticket of updating the help text?

**Review by sabrinang — APPROVED** [review]
Thanks for whitelisting it for it to preview. I also filed https://github.com/mozilla/foundation.mozilla.org/issues/6256 for separate airtable block updates.

**Pull request merged**